faq/faq.rst

```diff
@@ -45,12 +46,11 @@ Are there any PostgreSQL features not supported by Citus?
 
 Since Citus provides distributed functionality by extending PostgreSQL, it uses the standard PostgreSQL SQL constructs. It provides full SQL support for queries which access a single node in the database cluster. These queries are common, for instance, in multi-tenant applications where different nodes store different tenants (see :ref:`when_to_use_citus`).
 
-Other queries which, by contrast, combine data from multiple nodes, do not support the entire spectrum of PostgreSQL features. However they still enjoy broad SQL coverage, including semi-structured data types (like jsonb, hstore), full text search, operators, functions, and foreign data wrappers. Note that the following constructs aren't supported natively for cross-node queries:
+Other queries which, by contrast, combine data from multiple nodes, do not support the entire spectrum of PostgreSQL features. However they still enjoy broad SQL coverage, including semi-structured data types (like jsonb, hstore), full text search, operators, functions, foreign data wrappers, and in many cases even transactional semantics. Note that the following constructs aren't supported natively for cross-node queries:
 
 * Window Functions
 * CTEs
```

Comment:
This will be changing as of 7.2. We should also probably amend this list a bit. We support window functions in certain scenarios now with subquery pushdown.

Reply:
Maybe we should leave this as-is on the current branch and update the section separately for 7.2, that way all the changes of this PR can be cherry-picked to 7.1

Reply:
I'm OK to update it in #571. We'll probably want to rework this FAQ in some good detail based on that.

Reply:
Yep, the recursive planning docs PR updates this question. I'll rebase away the change here when the other PR is merged.